### DISCUSSION TITLE
Folder Monitor

### DISCUSSION THREAD

**Discussion by voska7** [RandomEngy/VidCoder#1029] [Ideas]
can you please consider to add a "folder watch monitor" feature? if a file is move to folder, vidcoder will add the file to default preset and endcode it.

**Comment by RandomEngy** [comment]
It's on the backlog. Actually, it's the most requested feature at this point. It's just going to take a lot of effort to implement. I'll just need to find a big chunk of time to do it.

**Comment by RandomEngy** [comment]
Added a folder watcher in 8.4 Beta.
